app/design/frontend/Magento/elogic-theme/web/css/source/_variables.less

```diff
@@ -20,6 +28,19 @@
 @font-family-name__base: @font-family-poppins;
 @font-family__base: @font-family-name__base, @font-family__sans-serif;
 
+//  Responsive breakpoints
+
+@screen__m_s: 480px;
```

Comment:
`@screen__xs: 480px` variable exists in:
code/lib/web/css/source/lib/variables/_responsive.less